pkg/runtime/oci/config/solaris.go

```diff
@@ -0,0 +1,3 @@
+// +build solaris
```

Comment:
Do we need to include Solaris and Windows files? We're not planning on supporting containers on these systems, strictly Linux. Is it necessary for other reasons to have this here?

Reply:
Indeed, it's not necessary, I just split OCI spec between platform to make things clearer and easier